### PR TITLE
Dynamic-Create-ENIs

### PR DESCRIPTION
**Pull request by ndrest-amzn**
First commit

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ndrest-amzn** [comment]
Committing for assessment and inclusion into repo

File: aws/solutions/Dynamic-Create-ENIs/Dynamic-Create-ENIs.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+            "DependsOn": [
+                "IamRoleLambda"
+            ],
+            "Properties": {
+                "Code": {
+                    "ZipFile": {
+                        "Fn::Join": [
+                            "\n",
+                            [
+                                "var AWS = require('aws-sdk');",
```
Comment (stilvoid) [inline aws/solutions/Dynamic-Create-ENIs/Dynamic-Create-ENIs.json]: Given this big chunk of code, this template might be better as yaml to make this bit more readable.

Comment (stilvoid) [reply aws/solutions/Dynamic-Create-ENIs/Dynamic-Create-ENIs.json]: Or even better, store the source in a separate file and instruct use of `aws cloudformation package` in the README.

**Comment by ndrest-amzn** [comment]
Thanks, I've converted the template into YAML.
Concerning the package command, I'd prefer to, but in this case it's to be used for a Knowledge base article and I'd like to demonstrate the full template with the custom resource code  to help user's understand how it all fits together (without needing to dig into source files), It's a demonstration mostly.

**Pull request closed without merging**